src/analysis/normed_space/operator_norm.lean

```diff
@@ -900,6 +900,14 @@ continuous_linear_equiv.uniform_embedding
   continuous_inv_fun := h₂,
   .. e }
 
+/-- Construct a continuous linear equivalence from a linear equivalence together with
+bounds in both directions. -/
+def linear_equiv.to_continuous_linear_equiv_of_bounds (e : E ≃ₗ[𝕜] F) (C_to C_inv : ℝ)
+  (h_to : ∀ x, ∥e x∥ ≤ C_to * ∥x∥) (h_inv : ∀ x : F, ∥e.symm x∥ ≤ C_inv * ∥x∥) : E ≃L[𝕜] F :=
+{ to_linear_equiv := e,
+  continuous_to_fun := linear_map.continuous_of_bound e.to_linear_map C_to h_to,
```

Comment:
```suggestion
  continuous_to_fun := e.continuous_of_bound C_to h_to,
```
does this work?  likewise below

Reply:
It doesn't: written this way Lean looks for `linear_equiv.continuous_of_bound`, which doesn't exist.

Reply:
Can you still do `e.to_linear_map.continuous_of_bound`?

Reply:
Yes, that works!